### DISCUSSION TITLE
container-app-conf necessary?

### DISCUSSION THREAD

**Discussion by rlukss** [femueller/python-n26#116] [General]
Thanks for building the application! Is the container-app-conf dependancy necessary? Wouldn't it be easier to just use the .env?

Cheers

**Comment by markusressel** [comment]
container-app-conf doesn't just read environment variables, it also provides YAML and TOML support, as well as a bunch of other things. So its not like it can be removed without breaking current functionality.

**Reply by rlukss** [reply]
Thanks for the reply!
